server/datastore/mysql/teams_test.go

```diff
@@ -590,6 +590,9 @@ func testTeamsMDMConfig(t *testing.T, ds *Datastore) {
 						BootstrapPackage:    optjson.SetString("bootstrap"),
 						MacOSSetupAssistant: optjson.SetString("assistant"),
 					},
+					WindowsSettings: fleet.WindowsSettings{
+						CustomSettings: optjson.SetSlice[string]([]string{"foo", "bar"}),
```

Comment:
Nit and out of curiosity, shouldn't that work without the `[string]` type parameter?

Reply:
you're absolutely right! thanks!